docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -181,6 +184,29 @@ New:
 zeebeGateway:
 ```
 
+#### Enabling External Elasticsearch
+
+It is possible to use external Elastichsearch. For more information on how to setup external Elasticsearch, please refer to the [Using Existing Elasticsearch](./guides/using-existing-elasticsearch.md) setup guide.
+
+##### Elasticsearch - values file
+
+The `global.elasticsearch.disableExporter` field has been deprecated in favour of `global.elasticsearch.enabled`. When `global.elasticsearch.enabled` is set to false, all configurations for Elasticsearch in all components are removed.
+
+The `global.elasticsearch.url` field has changed. If you are using the default values.yaml and have not configured the URL, then no change is required. On the other hand, if the URL value is used, then instead of specifying a single URL, you must now explicitly specify the Protocol, Host, and Port seperately like so:
```

Comment:
We also need to mention the other changed keys `global.elasticsearch.protocol`, `global.elasticsearch.host`, and `global.elasticsearch.port`.

Reply:
done